environments.json

```diff
@@ -6204,6 +6204,19 @@
     "short": "JrS 2.4.0",
     "platformtype": "engine",
     "release": "2021-01-22",
+    "obsolete": true,
```

Comment:
This is the current release, so it should be `"obsolete": false,`.  All the previous versions are more than 5 years old now, they should be `"obsolete": "very",`

Reply:
3.0 was released, so `"obsolete": true` is correct now.